### PR TITLE
fix(store): add the missing bracket

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

~Remove `true && ` which is always `true`.~

add the bracket 

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: modules/store/src/meta-reducers/immutability_reducer.ts [hunk 1]
```diff
@@ -20,8 +20,7 @@ function freeze(target: any) {
     if (
       hasOwnProperty(target, prop) && targetIsFunction
         ? prop !== 'caller' && prop !== 'callee' && prop !== 'arguments'
-        : true &&
-          (isObjectLike(propValue) || isFunction(propValue)) &&
+        : (isObjectLike(propValue) || isFunction(propValue)) &&
```
Comment (timdeschryver) [inline modules/store/src/meta-reducers/immutability_reducer.ts]: Seems like prettier in combination with myself messed this up, it should be:


```
function freeze(target: any) {
  Object.freeze(target);

  const targetIsFunction = isFunction(target);

  Object.getOwnPropertyNames(target).forEach(prop => {
    const propValue = target[prop];
    if (
      hasOwnProperty(target, prop) &&
      (targetIsFunction
        ? prop !== 'caller' && prop !== 'callee' && prop !== 'arguments'
        : true) &&
      (isObjectLike(propValue) || isFunction(propValue)) &&
      !Object.isFrozen(propValue)
    ) {
      freeze(propValue);
    }
  });

  return target;
```

Reference: [deep-freeze-strict](https://github.com/jsdf/deep-freeze/blob/master/index.js)

Comment (alex-okrushko) [reply modules/store/src/meta-reducers/immutability_reducer.ts]: Done.

**Comment by timdeschryver** [comment]
For full coverage a change the invoke function at https://github.com/ngrx/platform/blob/27c36687b18d34b559b59539d242bb4ba595cae5/modules/store/spec/meta-reducers/immutability_reducer.spec.ts#L22

To be
```
function invokeReducer(reduce: Function) {
      immutabilityCheckMetaReducer((state, action) => {
        reduce(action);
        return state;
      })({}, { type: 'invoke', numbers: [1, 2, 3], fun: function() {} });
}
```

**Comment by alex-okrushko** [comment]
Done.

**Pull request merged**